src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -126,6 +126,7 @@
     public static boolean logCascadingWorldGeneration = true; // see Chunk#logCascadingWorldGeneration()
     public static boolean fixVanillaCascading = false; // There are various places in vanilla that cause cascading worldgen. Enabling this WILL change where blocks are placed to prevent this.
                                                        // DO NOT contact Forge about worldgen not 'matching' vanilla if this flag is set.
+    public static boolean flushNetworkOnTick = false; //because vanilla spmms TCP packets
```

Comment:
```suggestion
    public static boolean flushNetworkOnTick = false; // send TCP packets on each tick instead of immediately (the vanilla default). setting this true reduce network overhead at the cost of some latency. https://github.com/MinecraftForge/MinecraftForge/pull/5126
```